security/rbac/delta_configs/connect-avro-distributed.properties.delta

```diff
@@ -52,7 +52,6 @@ config.providers=secret
 config.providers.secret.class=io.confluent.connect.secretregistry.rbac.config.provider.InternalSecretConfigProvider
 config.providers.secret.param.master.encryption.key=password1234
 
-config.providers.secret.param.kafkastore.topic=_secrets
```

Comment:
Just double-checking--you've verified that removing this property from the config provider's configuration doesn't break things, right?

Reply:
@C0urante correct.

Reply:
👍